Block/Customer/Savedcard.php

```diff
@@ -38,7 +38,7 @@ public function getSavedCards()
         $customerCards = $this->getCustomerCards();
         if ($customerCards && $customerCards->count()) {
             foreach ($customerCards as $card) {
-                $cards[]=['type'=>$card->getMethod(),'number'=>$card->getIdentifier(),'contract_id'=>$card->getId()];
+                $cards[]=['type'=>$card->getMethod(),'number'=>$card->getIdentifier(),'contract_id'=>$card->getId(), 'currency'=>$card->getcurrency()];
```

Comment:
Nit pick, but could we format this array to break each key-value pair into its own line

ex.

```
[
    'type' => $card->getMethod(),
    'number'=>$card->getIdentifier(),
    ...
]
```